config/src/config/test_data/validator_full_node.yaml

```diff
@@ -32,3 +32,7 @@ full_node_networks:
       seed_peers:
           "c227da54069989f283712e4016704660":
               - "/ip4/127.0.0.1/tcp/58259/ln-noise-ik/c998dcd54c3daf48e0ad516d94b7be0b0b7a27caa00541f2b2c14b13500df10b/ln-handshake/0"
+
+upstream:
+    networks:
+        - private: vfn
```

Comment:
is this public network the one advertised on-chain or the one that will be used to fallover to other on-chain-advertised VFN's? how are we differentiating b/w these two public networks (because one of them has to be defined in the upstream config)